mailpoet/assets/js/src/newsletters/types.tsx

```diff
@@ -36,9 +36,7 @@ export function NewsletterTypes({
   const [isCreating, setIsCreating] = useState(false);
 
   const [isSelectEditorModalOpen, setIsSelectEditorModalOpen] = useState(false);
-  const isNewEmailEditorEnabled =
-    MailPoet.FeaturesController.isSupported('gutenberg_email_editor') &&
-    window.mailpoet_block_email_editor_enabled;
+  const isNewEmailEditorEnabled = window.mailpoet_block_email_editor_enabled;
```

Comment:
I think this can be removed, and then we can also remove the window property `mailpoet_block_email_editor_enabled`. It is not critical. Or is there some reason to keep it?
From what I have checked, it only affects rendering a button, which should not be conditional anymore.

Reply:
We use the conditional to check if we can show the email editor button. The [variable](https://github.com/mailpoet/mailpoet/blob/1103e5fe49cf8f959139dced5c55e5409b2ac747/mailpoet/lib/AdminPages/Pages/Newsletters.php#L173) is set (or disabled) when the email editor [requirements aren't met](https://github.com/mailpoet/mailpoet/blob/1103e5fe49cf8f959139dced5c55e5409b2ac747/packages/php/email-editor/src/Engine/class-dependency-check.php#L23-L28).

Reply:
Ah. This is something that was added or changed. I guess we didn't initially check the requirement, and it depended only on the feature flag. Thanks for the explanation 👍